sway/commands/move.c

```diff
@@ -256,7 +256,25 @@ static void container_move_to_container(struct sway_container *container,
 	if (destination->view) {
 		container_add_sibling(destination, container, 1);
 	} else {
-		container_add_child(destination, container);
+		/* For split containers, we use the currently focused container within it.
+		* This allows setting marks on, e.g., tabbed containers which will move
+		* con to a new tab behind the focused tab. */
+		sway_log(SWAY_DEBUG, "target is a split container, descending to the currently focused child.");
+		struct sway_node *focused = seat_get_active_tiling_child(
+				config->handler_context.seat, &destination->node);
+		if (!focused || focused == &destination->node) {
+			// The container has no children
+			container_add_child(destination, container);
+			return;
+		}
+		struct sway_container *target = focused->sway_container;
+		int index = container_sibling_index(target) + 1;
+		if (target->pending.parent) {
+			container_insert_child(target->pending.parent, container, index);
+		} else {
+			// TODO: This depends on missing support to mark workspaces.
```

Comment:
My swat at next steps here are to file an i3-compat issue for making workspaces markable and link it here.

In the meantime, I'm cooking up a workaround for folks who are interested in this.

Reply:
Updated TODO to reference an issue for tracking.